alpaca/entities.go

```diff
@@ -175,9 +175,12 @@ type Fundamental struct {
 }
 
 type CalendarDay struct {
-	Date  string `json:"date"`
-	Open  string `json:"open"`
-	Close string `json:"close"`
+	Date           string `json:"date"`
+	Open           string `json:"open"`
+	Close          string `json:"close"`
+	SessionOpen    string `json:"session_open"`
+	SessionClose   string `json:"session_close"`
```

Comment:
I wouldn't like to add the session open/close fields here as they were exposed to the API unintentionally and there is no good use for them either as they don't really mean much as far as trading goes.

We can't remove it from the API response as that would be a breaking change, but at least we avoid adding it here to avoid more breaking changes in future.

Reply:
Can you please add this comment to the original issue: https://github.com/alpacahq/alpaca-trade-api-go/issues/266?